### PR TITLE
[ACCEL] Add assessments list for staff review

### PR DESCRIPTION
**Pull request by arsentur**
**Description:**
- Change methods that get students assessments for staff review. (Now it returns all submissions for staff review)
- Delete useless tests and fix another
- Create js handlers for showing staff area and change templates

**YouTrack:**
[ACCEL-432](https://youtrack.raccoongang.com/issue/ACCEL-432)

### CODE REVIEW

File: openassessment/assessment/models/staff.py [hunk 1]
```diff
@@ -80,58 +83,35 @@ def get_workflow_statistics(cls, course_id, item_id):
...
+    def get_submission_for_review(cls, course_id, item_id):
         """
-        Find a submission for staff assessment. This function will find the next
-        submission that requires assessment, excluding any submission that has been
-        completely graded, or is actively being reviewed by other staff members.
+        Find all submissions for staff assessment.
 
         Args:
-            submission_uuid (str): The submission UUID from the student
+            course_id (str): The submission UUID from the student
```
Comment (flying-pi) [inline openassessment/assessment/models/staff.py]: Is course_id is submission uuid? It little bit confused.

File: openassessment/assessment/api/staff.py [hunk 2]
```diff
@@ -249,36 +248,37 @@ def get_submission_to_assess(course_id, item_id, scorer_id):
...
             'submitted_at': datetime.datetime(2014, 1, 29, 23, 14, 52, 649284, tzinfo=<UTC>),
             'created_at': datetime.datetime(2014, 1, 29, 17, 14, 52, 668850, tzinfo=<UTC>),
             'answer': { ... }
-        }
+        }, ...]
 
     """
-    student_submission_uuid = StaffWorkflow.get_submission_for_review(course_id, item_id, scorer_id)
+
+    student_submission_uuid = StaffWorkflow.get_submission_for_review(course_id, item_id)
```
Comment (flying-pi) [inline openassessment/assessment/api/staff.py]: Could you add plural part in the variable name?

File: openassessment/assessment/api/staff.py [hunk 3]
```diff
@@ -228,16 +228,15 @@ def get_assessment_scores_by_criteria(submission_uuid):
...
-    Retrieves a submission for assessment for the given staff member.
+    Retrieves a list of submissions for assessment for the given staff member.
 
     Args:
         course_id (str): The course that we would like to fetch submissions from.
         item_id (str): The student_item (problem) that we would like to retrieve submissions for.
-        scorer_id (str): The user id of the staff member scoring this submission
 
     Returns:
         dict: A student submission for assessment. This contains a 'student_item',
```
Comment (flying-pi) [inline openassessment/assessment/api/staff.py]: Seems this function returns list now*

File: openassessment/xblock/static/js/src/lms/oa_staff_area.js [hunk 4]
```diff
@@ -139,6 +139,50 @@
             return deferred.promise();
         },
 
+        /**
+         * Upon request, sets up handlers for collapsing section of the staff area.
+         */
+        setUpStaffAreaCollapsing: function () {
+            $('.ui-accordion-header').on('click', function () {
```
Comment (flying-pi) [inline openassessment/xblock/static/js/src/lms/oa_staff_area.js]: As for me, better, when this function is decomposed. It is about, that there are several independent functions with a name such as  `closeAssesment` or `toggleAssesment` and bellows this function is marked as some event listener (` $('.ui-accordion-header').on('click',toggleAssesment)`).

File: openassessment/assessment/models/staff.py (hunk 1 above)
Comment (arsentur) [reply openassessment/assessment/models/staff.py]: Done

File: openassessment/assessment/api/staff.py (hunk 2 above)
Comment (arsentur) [reply openassessment/assessment/api/staff.py]: Done

File: openassessment/assessment/api/staff.py (hunk 3 above)
Comment (arsentur) [reply openassessment/assessment/api/staff.py]: Done

File: openassessment/xblock/static/js/src/lms/oa_staff_area.js (hunk 4 above)
Comment (arsentur) [reply openassessment/xblock/static/js/src/lms/oa_staff_area.js]: Done

File: openassessment/assessment/models/staff.py [hunk 5]
```diff
@@ -80,58 +83,32 @@ def get_workflow_statistics(cls, course_id, item_id):
...
-                requesting a submission for assessment. This is used to explicitly
-                avoid giving the student their own submission, and determines the
-                associated Peer Workflow.
+            course_id (str): The course ID for filtering submissions for current course.
             item_id (str): The student_item that we would like to retrieve submissions for.
-            scorer_id (str): The user id of the staff member scoring this submission
 
         Returns:
-            submission_uuid (str): The submission_uuid for the submission to review.
+            list: The list of submission_uuid's for review.
```
Comment (idegtiarov) [inline openassessment/assessment/models/staff.py]: Not a list but a queryset with a list of values. Does it is what is expected?

File: openassessment/assessment/models/staff.py [hunk 6]
```diff
@@ -80,58 +83,32 @@ def get_workflow_statistics(cls, course_id, item_id):
...
-            workflow = staff_workflows[0]
-            workflow.scorer_id = scorer_id
-            workflow.grading_started_at = now()
-            workflow.save()
-            return workflow.submission_uuid
+            ).values_list('submission_uuid', flat=True)
+
         except DatabaseError:
             error_message = (
                 u"An internal error occurred while retrieving a submission for staff grading"
```
Comment (idegtiarov) [inline openassessment/assessment/models/staff.py]: typo: s/a submission/submissions/

**Pull request merged**